pysatModels/models/sami2py_sami2.py

```diff
@@ -0,0 +1,198 @@
+    # rename time variable to be pysat compatible
+    data = data.rename({'ut': 'time'})
+    # move attributes to the Meta object
+    # these attributes will be trasnferred to the Instrument object
+    # automatically by pysat
+    meta = pysat.Meta()
+    for attr in data.attrs:
+        setattr(meta, attr[0], attr[1])
+    data.attrs = []
+
+    # fill Meta object with variable information
+    for key in data.variables.keys():
+        attrs = data.variables[key].attrs
+        meta[key] = attrs
+
+    return data, meta
+
+
+def download(date_array=None, tag=None, sat_id=None, data_path=None, user=None,
+             password=None, **kwargs):
+    """Downloads sami2py data.  Currently only retrieves test data from github
+
+    Parameters
+    ----------
+    date_array : array-like
+        list of datetimes to download data for. The sequence of dates need not
+        be contiguous.
+    tag : string ('')
+        Tag identifier used for particular dataset. This input is provided by
+        pysat.
+    sat_id : string  ('')
+        Satellite ID string identifier used for particular dataset. This input
+        is provided by pysat.
+    data_path : string (None)
+        Path to directory to download data to.
+    user : string (None)
+        User string input used for download. Provided by user and passed via
+        pysat. If an account
+        is required for dowloads this routine here must error if user not
+        supplied.
+    password : string (None)
+        Password for data download.
+    **kwargs : dict
+        Additional keywords supplied by user when invoking the download
+        routine attached to a pysat.Instrument object are passed to this
+        routine via kwargs.
+
+    Notes
+    -----
+    This routine is invoked by pysat and is not intended for direct use by
+    the end user.
+
+    The test object generates the datetime requested by the user, which may not
+    match the date of the model run.
+
+    """
+
+    if tag == 'test':
+        date = date_array[0]
+        remote_url = 'https://github.com/sami2py/sami2py/'
+        remote_path = 'blob/main/sami2py/tests/test_data/'
+        # Need to tell github to show the raw data, not the webpage version
+        fname = 'sami2py_output.nc?raw=true'
+        # Use pysat-compatible name
+        format_str = supported_tags[sat_id][tag]
+        saved_local_fname = os.path.join(data_path,
+                                         format_str.format(year=date.year,
+                                                           month=date.month,
+                                                           day=date.day))
+        remote_path = '/'.join((remote_url.strip('/'), remote_path.strip('/'),
+                                fname))
+        req = requests.get(remote_path)
+        if req.status_code != 404:
+            open(saved_local_fname, 'wb').write(req.content)
+
+    else:
+        warnings.warn('Not implemented in this version.')
```

Comment:
Perhaps change warning to indicate this is only intended to be used for accessing the test files?